### PR TITLE
Create infix-to-postfix.c

### PR DESCRIPTION
**Pull request by parizadas**

### CODE REVIEW

File: infix-to-postfix.c [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#include<stdio.h>
+#include<string.h>
```
Comment (nimishbongale) [inline infix-to-postfix.c:2]: You may want to put this file in a folder inside Task 3 subdirectory. Please have a look at the `README.md` file. 

Please also add a small description & sample input/output cases in a multiline comment on the top.

**Review by nimishbongale — COMMENTED** [review]
Good work @parizadas!

Small changes, I'm sure you can do them!

File: infix-to-postfix.c [hunk 2]
```diff
@@ -0,0 +1,114 @@
+#include<stdio.h>
```
Comment (parizadas) [inline infix-to-postfix.c:1]: \\infix to postfix conversion

File: infix-to-postfix.c [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+char stack[50];
+int top=-1;
+void post(char infix[]);
+void push(char);
+char pop();
+
+void main()
+{
+    char infix[25];
+    printf("\nEnter the infix expression : ");
```
Comment (parizadas) [inline infix-to-postfix.c:12]: Entering an infix expression.

File: infix-to-postfix.c [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    gets(infix);
+    post(infix);
+    getch();
+}
+
+void push(char symb)
+{
+    if(top>=49)
+    {
+          printf("\nStack Overflow");
```
Comment (parizadas) [inline infix-to-postfix.c:22]: Check overflow condition

File: infix-to-postfix.c [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+         getch();
+         return;
+    }
+    else
```
Comment (parizadas) [inline infix-to-postfix.c:26]: Push an element

File: infix-to-postfix.c [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    {
+          top=top+1;
+         stack[top]=symb;
+    }
+}
+char pop()
+{
+     char item;
+     if(top==-1)
```
Comment (parizadas) [inline infix-to-postfix.c:35]: To check whether the stack is empty or not

**Comment by nimishbongale** [comment]
@parizadas 

You have to make these changes within your code. You don't have to necessarily review your own code.

File: infix-to-postfix.c [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+     {
+            printf("\nStack is empty");
+            getch();
+            return(0);
+      }
+      else
```
Comment (parizadas) [inline infix-to-postfix.c:41]: To delete an element

File: infix-to-postfix.c (hunk 1 above)
Comment (nimishbongale) [reply infix-to-postfix.c:2]: Please move your file into a folder named `InfixToPostfix`, under `Task 3` folder.

File: infix-to-postfix.c [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+     {
+            item=stack[top];
+            top--;
+     }
+     return(item);
+}
+int preced(char ch)
```
Comment (parizadas) [inline infix-to-postfix.c:48]: For checking the precedence of symbols

File: infix-to-postfix.c [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+            return(4);
+      }
+      else if(ch==43)
+      {
+             return(3);
+      }
+      else
+      return(2);
+}
+void post(char infix[])
```
Comment (parizadas) [inline infix-to-postfix.c:65]: For converting infix into postix

**Comment by parizadas** [comment]
> @parizadas
> 
> You have to make these changes within your code. You don't have to necessarily review your own code.

Okay... sorry.. Actually I am new to github..

**Comment by nimishbongale** [comment]
> Okay... sorry.. Actually I am new to github..

That is totally okay, we have this repo for that purpose! You can learn and explore this place 😄 

Always read the README.md file before contributing! Good luck!

I might close this PR so that you can start afresh. I hope that's okay! 😄

**Pull request closed without merging**